pkg/agent/types/net.go

```diff
@@ -28,3 +28,11 @@ var (
 	// SNAT IP for a "Pod -> external" egress packet, that is to be SNAT'd.
 	SNATIPMarkMask = uint32(0xFF)
 )
+
+// IP Route tables
+const (
+	// MinEgressRouteTable to MaxEgressRouteTable are the route table IDs that can be configured on a Node for Egress traffic.
+	// Each distinct subnet uses one route table. 20 subnets should be enough.
```

Comment:
Probably first define a constant for the maximum number of VLANs/subnets.

Reply:
but then it may less clear the range of the tables used by Egress? I was thinking we can easily figure out which tables are used for from the ranges. Like:
```
MinXXXRouteTable = 101
MaxXXXRouteTable = 120

MinYYYRouteTable = 121
MaxYYYRouteTable = 130
```
And the id allocator takes min and max parameters at the moment.

Reply:
No strong opinion here. At least in the error log, I hope we can add the maximum limit for users to know.

Reply:
Added an error when it fails to allocate subnet:
```
fmt.Errorf("error allocating table for subnet %v due to exceeding max allowed subnets %d: %w", subnetInfo, maxSubnetsPerNodes, err)
```
And egress.md has documented the limitation.